### PR TITLE
post blog of release of v1.1.4

### PR DESCRIPTION
**Pull request by lunny**
As title.

### CODE REVIEW

**Comment by lafriks** [comment]
LGTM

File: content/post/release-of-1.1.4.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+<!--more-->
+
+## Changelog
+
+* BUGFIXES
+  * Fix rendering of external links (#2292) (#2315)
+  * Fix deleted milestone bug (#1942) (#2300)
+  * fix 500 error when view an issue which's milestone deleted (#2297) (#2299)
+  * Fix SHA1 hash linking (#2143) (#2293)
+  * back port from #1709 (#2291)
```
Comment (bkcsoft) [inline content/post/release-of-1.1.4.md]: This is a security-fix (code injection), not a bugfix...

Comment (strk) [reply content/post/release-of-1.1.4.md]: Also I think the release notes should be self-explanatory, wherease `back port from #1709 (#2291)` is an obscure message

Comment (lafriks) [reply content/post/release-of-1.1.4.md]: These are auto generated so PR should have correct title/labels but otherwise I agree

**Comment by lunny** [comment]
@bkcsoft @strk @lafriks done.

**Comment by lunny** [comment]
@bkcsoft @strk @lafriks done.

**Comment by strk** [comment]
LGTM

**Comment by sapk** [comment]
Let Get This Merge ^^

**Pull request merged**